src/deadline/client/api/_attachment.py

```diff
@@ -0,0 +1,82 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+import os
+import boto3
+import json
+
+from contextlib import ExitStack
+from typing import Optional, List, Dict
+
+from deadline.job_attachments.asset_manifests.base_manifest import BaseAssetManifest
+from deadline.job_attachments.asset_manifests.decode import decode_manifest
+from deadline.job_attachments.download import download_files_from_manifests
+from deadline.job_attachments.models import JobAttachmentS3Settings, PathMappingRule
+from deadline.job_attachments.progress_tracker import DownloadSummaryStatistics
+from deadline.client.exceptions import NonValidInputError
+
+
+def attachment_download(
+    manifests: List[str],
+    s3_root_path: str,
```

Comment:
sorry, I think this should be s3_cas_path, which is `s3://bucket-name` and `cas/path/root`